airbyte-webapp/src/core/domain/connector/types.ts

```diff
@@ -4,6 +4,15 @@ import {
 } from "core/domain/connection";
 import { DestinationSyncMode } from "core/domain/catalog";
 
+export enum releaseStage {
```

Comment:
```suggestion
export enum ReleaseStage {
```

Types, enums, classes, should be written uppercase in the beginning